docs/examples/code-cells.ipynb

```diff
@@ -0,0 +1,960 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For comparison,\n",
+    "this is how it would look in PNG format ..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%config InlineBackend.figure_formats = ['png']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "... and in `'png2x'` (a.k.a. `'retina'`) format:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%config InlineBackend.figure_formats = ['png2x']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Pandas Dataframes\n",
+    "\n",
+    "[Pandas dataframes](https://pandas.pydata.org/pandas-docs/stable/user_guide/dsintro.html#dataframe)\n",
+    "should be displayed as nicely formatted HTML tables (if you are using HTML output)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.DataFrame(\n",
+    "    np.random.randint(0, 100, size=[10, 4]),\n",
+    "    columns=[r\"$\\alpha$\", r\"$\\beta$\", r\"$\\gamma$\", r\"$\\delta$\"],\n",
+    ")\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Xarray Datasets\n",
+    "\n",
+    "[Xarray datasets](https://docs.xarray.dev/en/stable/generated/xarray.Dataset.html)\n",
+    "should also be shown in their beautiful HTML representation.\n",
+    "LaTeX/PDF output will fall back to the plain text representation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import xarray as xr"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = {}\n",
+    "for name in [\"t\", \"q\"]:\n",
+    "    da = xr.DataArray(\n",
+    "        [\n",
+    "            [[11, 12, 13], [21, 22, 23], [31, 32, 33]],\n",
+    "            [[14, 15, 16], [24, 25, 26], [34, 35, 36]],\n",
+    "        ],\n",
+    "        coords={\"level\": [\"500\", \"700\"], \"x\": [1, 2, 3], \"y\": [4, 5, 6]},\n",
+    "        name=name,\n",
+    "        attrs={\"param\": name},\n",
+    "    )\n",
+    "    a[name] = da\n",
+    "\n",
+    "ds = xr.Dataset(a, attrs={\"title\": \"Demo dataset\", \"year\": 2024})\n",
+    "ds"
+   ]
```

Comment:
output is illegible in dark mode, though for xarray content this is a known bug, not a regression (#2189)

Reply:
Xarray in dark mode looks better now. Screenshot from preview build:

<img width="712" alt="" src="https://github.com/user-attachments/assets/c0342815-1d73-48b7-b60e-965854f54544" />